### PR TITLE
cudatext, don't depend on python library, add cmd:python3 for it

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: app-editors/cudatext/cudatext-1.211.3.0.recipe [hunk 1]
```diff
@@ -54,8 +54,8 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
+	cmd:python3
```
Comment (Begasus) [inline app-editors/cudatext/cudatext-1.211.3.0.recipe:57]: @OscarL  Would this be OK? ;)

Comment (Begasus) [reply app-editors/cudatext/cudatext-1.211.3.0.recipe]: Done, thanks!

File: app-editors/cudatext/cudatext-1.211.3.0.recipe [hunk 2]
```diff
@@ -54,8 +54,8 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
+	cmd:python3 # indirect way of asking for any `lib:libpython3.xx` to be installed.
```
[the remark this answers is not in the document]
Comment (Begasus) [reply app-editors/cudatext/cudatext-1.211.3.0.recipe:57]: It actualy searches for "pylib__haiku": "libpython3.10.so.1.0", in the default settings file:

https://github.com/Alexey-T/CudaText/blob/master/app/settings_default/default.json#L1455

So maybe we can just leave the dependency out, it's not a hard requirement and should use default installed python.

[the remark this answers is not in the document]
Comment (korli) [reply app-editors/cudatext/cudatext-1.211.3.0.recipe:57]: IMO just declare the lib:libpython3.xx version you tested, if it's a hard requirement. If not, leave it out.

**Pull request merged**